cpuid/src/cpu_leaf.rs

```diff
@@ -225,3 +200,73 @@ pub mod leaf_0xb {
         pub const LEVEL_NUMBER_BITRANGE: BitRange = bit_range!(7, 0);
     }
 }
+
+pub mod leaf_0x80000000 {
+    pub const LEAF_NUM: u32 = 0x8000_0000;
+
+    pub mod eax {
+        use bit_helper::BitRange;
+
+        pub const LARGEST_EXTENDED_FN_BITRANGE: BitRange = bit_range!(31, 0);
+    }
+}
+
+pub mod leaf_0x80000001 {
+    pub const LEAF_NUM: u32 = 0x8000_0001;
+
+    pub mod ecx {
+        pub const LZCNT_SHIFT: u32 = 5; // advanced bit manipulation
+        pub const PREFETCH_SHIFT: u32 = 8; // 3DNow! PREFETCH/PREFETCHW instructions
+        pub const TOPOEXT_INDEX: u32 = 22;
+    }
+
+    pub mod edx {
+        pub const PDPE1GB_SHIFT: u32 = 26; // 1-GByte pages are available if 1.
+    }
+}
+
+pub mod leaf_0x80000008 {
+    pub const LEAF_NUM: u32 = 0x8000_0008;
+
+    pub mod ecx {
+        use bit_helper::BitRange;
+
+        // The number of bits in the initial ApicId value that indicate thread ID within a package
+        pub const THREAD_ID_SIZE_BITRANGE: BitRange = bit_range!(15, 12);
+        // The number of threads in the package - 1
+        pub const NUM_THREADS_BITRANGE: BitRange = bit_range!(7, 0);
+    }
+}
+
+// Extended Cache Topology Leaf
+pub mod leaf_0x8000001d {
+    pub const LEAF_NUM: u32 = 0x8000_001d;
+
+    // inherit eax from leaf_cache_parameters
+    pub use cpu_leaf::leaf_cache_parameters::eax;
+}
+
+// Extended Cache Topology Leaf
```

Comment:
Previous  section's comment is also "Extended Cache Topology Leaf". Is this how it is inteded to be?

Reply:
I forgot to change the comment. Fixed.